.test-infra/jenkins/job_LoadTests_Python.groovy

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def smokeTestConfigurations = [
+        [
+                title        : 'GroupByKey Python load test Direct',
+                itClass      : 'apache_beam.testing.load_tests.group_by_key_test:GroupByKeyTest.testGroupByKey',
+                runner       : CommonTestProperties.Runner.DIRECT_PYTHON,
+                jobProperties: [
+                        publish_to_big_query: true,
+                        project             : 'apache-beam-testing',
+                        metrics_dataset     : 'load_test_SMOKE',
+                        metrics_table       : 'python_direct_gbk',
+                        input_options       : '\'{"num_records": 100000,' +
+                                '"key_size": 1,' +
+                                '"value_size":1,' +
+                                '"bundle_size_distribution_type": "const",' +
```

Comment:
nit/question (i don't know python sdk well): do we have default values for those options? since this suite is to verify if everything works (smoke) maybe we should use the default values there?

Reply:
Yes, there are. My idea behind specifying everything was to make somebody who would like simply check job configuration to have whole job parameters here without necessity to check the defaults of test/functionality implementation.